src/contracts/pods/EigenPod.sol

```diff
@@ -55,6 +56,13 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
 
     /// @notice The amount of eth, in wei, that is restaked per ETH validator into EigenLayer
     uint256 public immutable REQUIRED_BALANCE_WEI;
+    
+
+    ///@notice The maximum amount of ETH, in gwei, a validator can have staked in the beacon chain
+    // TODO: consider making this settable by owner
+    uint64 public immutable MAX_VALIDATOR_BALANCE_GWEI = 32e9;
```

Comment:
if you're setting these, they can be constants. I would make them immutable and set them in constructor

Reply:
Set them in here so I didn't have to change all the constructors in testing.  Intention is to have them set in the constructor for sure